operator/tests/e2e-v2/resource-redpanda-crud/00-assert-create-redpanda.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   name: redpanda
 spec:
   chartRef:
-    chartVersion: "5.3.2"
+    chartVersion: "5.9.10"
```

Comment:
Should we just leave all these empty for now? It'll default to the pinned chart version. That way we don't have to keep updating them.

Reply:
I'm fine dropping it.